### PR TITLE
[Cyber] Bump up copyright year

### PR DESCRIPTION
**Pull request by aaronhma**
This is bumping up the copyright year from 2017, 2018 to 2019 only in the ```cyber/``` folder.

### CODE REVIEW

**Comment by aaronhma** [comment]
@xiaoxq @natashadsouza @Capri2014 My 2nd PR as the youngest Apollo contributor at 11 years old. in this PR, I changed the ```cyber/``` folder's copyright version to 2019.

File: cyber/doxy-docs/source/CyberRT_Developer_Tools.md [hunk 1]
```diff
@@ -264,12 +264,12 @@ $ cyber_recorder record -a
 #### Replay a record file
 
 ```
-$ cyber_recorder play -f 20180720202307.record
-file: 20180720202307.record, chunk_number: 1, begin_time: 1532089398663399667, end_time: 1532089404688079759, message_number: 75
+$ cyber_recorder play -f 20190720202307.record
```
Comment (natashadsouza) [inline cyber/doxy-docs/source/CyberRT_Developer_Tools.md]: Please do not change the date on the bag files

File: cyber/doxy-docs/source/CyberRT_Developer_Tools.md [hunk 2]
```diff
@@ -264,12 +264,12 @@ $ cyber_recorder record -a
...
+file: 20190720202307.record, chunk_number: 1, begin_time: 1532089398663399667, end_time: 1532089404688079759, message_number: 75
 please wait for loading and playing back record...
 Hit Ctrl+C to stop replay, or Space to pause.
 [RUNNING]  Record Time: 1532089404.688080    Progress: 6.024680 / 6.024680
-play finished. file: 20180720202307.record
+play finished. file: 20190720202307.record
```
Comment (natashadsouza) [inline cyber/doxy-docs/source/CyberRT_Developer_Tools.md]: same as above

File: cyber/tools/cyber_recorder/main.cc [hunk 3]
```diff
@@ -118,11 +118,11 @@ void DisplayUsage(const std::string& binary, const std::string& command,
                   << " rate by FACTOR" << std::endl;
         break;
       case 'b':
-        std::cout << "\t-b, --begin <2018-07-01 00:00:00>\t" << command
+        std::cout << "\t-b, --begin <2019-07-01 00:00:00>\t" << command
```
Comment (natashadsouza) [inline cyber/tools/cyber_recorder/main.cc]: @quning18 confirm this change?

File: cyber/doxy-docs/source/CyberRT_Developer_Tools.md [hunk 4]
```diff
@@ -327,8 +327,8 @@ $ rosbag_to_record demo_2.5.bag demo.record
 record_file:    demo.record
 version:        1.0
 duration:       19.995227 Seconds
-begin_time:     2018-04-17 06:25:36
-end_time:       2018-04-17 06:25:55
+begin_time:     2019-04-17 06:25:36
+end_time:       2019-04-17 06:25:55
```
Comment (natashadsouza) [inline cyber/doxy-docs/source/CyberRT_Developer_Tools.md]: Revert

File: cyber/doxy-docs/source/CyberRT_Developer_Tools.md [hunk 5]
```diff
@@ -206,8 +206,8 @@ usage: cyber_recorder split [options]
     -o, --output <file>                output record file
     -a, --all                          all channels
     -c, --channel <name>               channel name
-    -b, --begin <2018-07-01 00:00:00>  begin at assigned time
-    -e, --end <2018-07-01 01:00:00>    end at assigned time
+    -b, --begin <2019-07-01 00:00:00>  begin at assigned time
```
Comment (natashadsouza) [inline cyber/doxy-docs/source/CyberRT_Developer_Tools.md]: Revert all above

File: cyber/doxy-docs/source/CyberRT_Developer_Tools.md [hunk 6]
```diff
@@ -228,8 +228,8 @@ $ cyber_recorder info demo.record
 record_file:    demo.record
 version:        1.0
 duration:       19.995227 Seconds
-begin_time:     2018-04-17 06:25:36
-end_time:       2018-04-17 06:25:55
+begin_time:     2019-04-17 06:25:36
```
Comment (natashadsouza) [inline cyber/doxy-docs/source/CyberRT_Developer_Tools.md]: same

File: cyber/doxy-docs/source/CyberRT_Developer_Tools.md [hunk 7]
```diff
@@ -264,12 +264,12 @@ $ cyber_recorder record -a
 #### Replay a record file
 
 ```
-$ cyber_recorder play -f 20180720202307.record
-file: 20180720202307.record, chunk_number: 1, begin_time: 1532089398663399667, end_time: 1532089404688079759, message_number: 75
+$ cyber_recorder play -f 20190720202307.record
+file: 20190720202307.record, chunk_number: 1, begin_time: 1532089398663399667, end_time: 1532089404688079759, message_number: 75
```
Comment (natashadsouza) [inline cyber/doxy-docs/source/CyberRT_Developer_Tools.md]: same as above

**Comment by natashadsouza** [comment]
@aaronhma thank you very much for supporting the Apollo dev community. We would like to retain the copyright license date to 2018 and also, our record bag formats have very specific dates on them that should not be changed. I am currently closing this pull request for the above mentioned reasons, but do continue your good work :)

**Pull request closed without merging**